### PR TITLE
adding job nonce to HolographBridge

### PR DESCRIPTION
**Pull request by ACC01ADE**
## Adding Job Nonce to Bridge Requests

We are adding job nonce to all bridge request to ensure that a unique jobPayloadHash is generated each time and makes it easier to catalog cross-chain requests individually.

- Adding _jobNonce to HolographBridge smart contract
- Implemented the use of nonce in all BridgeIn calls

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -167,11 +164,6 @@ const config: HardhatUserConfig = {
       chainId: networks.mumbai.chain,
       accounts: [MUMBAI_PRIVATE_KEY],
     },
-    fuji: {
```
Comment (alexanderattar) [inline hardhat.config.ts]: why was fuji removed here?

**Comment by alexanderattar** [comment]
Changes generally make sense to me

**Review by alexanderattar — APPROVED** [review]
lg

File: hardhat.config.ts (hunk 1 above)
Comment (ACC01ADE) [reply hardhat.config.ts]: oops, fixed it

**Pull request merged**